<launch>
<!--   <param name="~framerate" value="20" /> -->

  <!-- Camera control -->

  <remap from="image" to="/camera/image_raw"/>

  <node pkg="usb_cam" type="usb_cam_node" name="camera" />

  <!-- Robot control -->

  <node pkg="yamabros" type="spur_server" name="spur_server" />

  <node pkg="dich" type="steering" name="steering" required="true">
    <param name="path_steerings" value="steerings.txt" />
    <!-- <param name="ang_vel" value="0.1" /> -->
  </node>

  <node pkg="dich" type="replay" name="replay" required="true">
    <param name="path" value="teach.mpg" />
    <param name="queue" value="30" />
    <param name="shift_width" value="64" />
  </node>

  <node pkg="rosbag" type="record" name="record_bag" args="/cmd_vel /odom /tf /forward" />
                                                <!-- args="/cmd_vel /odom /tf /forward /camera/image_raw" -->
</launch>
